### PR TITLE
Get only links with specific type from GravitonFile

### PR DESCRIPTION
**Pull request by q-src**

### CODE REVIEW

**Comment by lapistano** [comment]
I'd really like to see our coding guidelines applied here as well (docblocks, etc)

**Comment by hairmare** [comment]
:+1: on @lapistano's remark

File: src/main/java/com/github/libgraviton/workerbase/model/GravitonFile.java [hunk 1]
```diff
@@ -3,25 +3,44 @@
...
     }
+
     public ArrayList<GravitonFileLinks> getLinks() {
         return links;
     }
+
+    public ArrayList<GravitonFileLinks> getLinks(String type) {
+        ArrayList<GravitonFileLinks> links = new ArrayList<GravitonFileLinks>();
+        for(GravitonFileLinks link : this.getLinks()) {
+            if(link.getType() == type) {
```
Comment (narcoticfresh) [inline src/main/java/com/github/libgraviton/workerbase/model/GravitonFile.java]: this should be `if(link.getType.equals(type))`, not `==` for testing equality (see [this](http://stackoverflow.com/questions/7520432/java-vs-equals-confusion)) @q-src

**Comment by narcoticfresh** [comment]
@lapistano indeed the `*.model*` subpackage had no javadoc as it consisted only of generated [POJOs](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) on base of Graviton resources. But people are now adding functions and stuff, it should be documented..

@q-src pls add comments, they are mostly generatable.. see the base package, all classes there are documented.

**Comment by q-src** [comment]
@narcoticfresh @lapistano I've just added some comments.
In addition, link comparison is now done with `equals()`.

**Pull request merged**